pkg/plugin/packetparser/_cprog/packetparser.c

```diff
@@ -221,21 +227,49 @@ static void parse(struct __sk_buff *skb, direction d)
 
 		p.src_port = udp->source;
 		p.dst_port = udp->dest;
+
+		flags = 1;
 	}
 	else
 	{
 		return;
 	}
 
-	bpf_perf_event_output(skb, &packetparser_events, BPF_F_CURRENT_CPU, &p, sizeof(p));
+	// Create a new conntrack key.
+	struct ct_v4_key key;
+	__builtin_memset(&key, 0, sizeof(struct ct_v4_key));
+	key.src_ip = p.src_ip;
+	key.dst_ip = p.dst_ip;
+	key.src_port = p.src_port;
+	key.dst_port = p.dst_port;
+	key.proto = p.proto;
+
+	// Process the packet in ct
+	bool report __attribute__((unused));
```

Comment:
What does 'unused' mean here why split out the declaration from assignment below?

Reply:
just to silence the compiler mostly, since we are using this var in an `ifdef` block (which we know for sure that it will be define lol)
![image](https://github.com/user-attachments/assets/1114312b-bfd2-482e-a844-d0894e9816bc)